fill/actuators/actuator.h

```diff
@@ -0,0 +1,17 @@
+#ifndef ACTUATOR_H
+#define ACTUATOR_H
+
+class Actuator {
+public: 
+    /* Actuates the actuator. 
```

Comment:
Can we refactor this file? The current implementation is very difficult to understand. Something like this maybe where active is open for valves or ignited for igniters?

// Starts actuation
virtual bool Actuate() = 0;

// Returns Active
virtual bool isActive() = 0

protected:
bool active;